rc/filetype/markdown.kak

```diff
@@ -59,9 +59,10 @@ add-highlighter shared/markdown/codeblock region -match-capture \
     ^(\h*)```\h*$ \
     fill meta
 
-add-highlighter shared/markdown/listblock region ^\h*[-*]\s ^\h*((?=[-*])|$) regions
-add-highlighter shared/markdown/listblock/marker region \A [-*]\s fill bullet
-add-highlighter shared/markdown/listblock/content default-region ref markdown/inline
+add-highlighter shared/markdown/listblock region ^\h*[-*]\s ^\h*((?=[-*])|$) group
+add-highlighter shared/markdown/listblock/regions regions
+add-highlighter shared/markdown/listblock/regions/ default-region ref markdown/inline
```

Comment:
Why does this introduce a region with only a default-region ? adding a `ref markdown/inline` to the group would have the same effect would'nt it ?

Reply:
The recent tendency to create regions for everything has triggered by habit of grouping all regions into a single group. I've made the change!